itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestWarehouseExternalDir.java

```diff
@@ -44,114 +44,94 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
-
+import org.apache.hadoop.hive.metastore.api.Database;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.apache.hadoop.hive.metastore.Warehouse;
 
-
-@org.junit.Ignore("HIVE-25266")
 public class TestWarehouseExternalDir {
-  private static final Logger LOG = LoggerFactory.getLogger(TestWarehouseExternalDir.class);
+  private  static final Logger LOG = LoggerFactory.getLogger(TestWarehouseExternalDir.class);
+
+  private static MiniHS2 miniHS2;
+  private static Hive db;
 
-  static MiniHS2 miniHS2;
-  static Hive db;
-  static Connection conn;
+  private static HiveConf conf;
+  private  static Connection conn;
```

Comment:
I would remove this field. 

1. `beforeTest()` creates a connection (line 88)
2. This is **_not closed_** but reassigned with a new connection in setUp() (line 119)
3. `tearDown()` closes the connection after every test (but there's only one test)

Even despite the lot of maintenance code this results in an unclosed connection. I would simply create and properly close local connection variables both in `beforeTest()` and `testExternalDefaultPaths()`

Reply:
Agree.
Removed unnecessary conn creation and closed connection in afterTest() class